lib/menu.php

```diff
@@ -305,6 +305,12 @@
                     'tip' => trans('Lists and reports printing'),
                     'prio' => 80,
                 ),
+                'VLAN List' => array(
```

Comment:
to jest tylko indeks - proszę o zmianę na `vlanlist`.

Reply:
https://github.com/lmsgit/lms/pull/1824/commits/9cef213df7507266818b9e675d77e1eb740a06de